python/GafferCyclesUI/CyclesAttributesUI.py

```diff
@@ -354,19 +354,29 @@ def __shaderSummary( plug ) :
 
 		# Shader
 
-		"attributes.useMis" : [
+		"attributes.emissionSamplingMethod" : [
```

Comment:
Do we want to provide any backwards compatibility for this change, by way of a config file that does some monkey patching? Do you know if Blender has maintained compatibility in some way?

Reply:
I *think* they name it differently internally, but yep we could monkey-patch our `cycles:shader:use_mis` to set the corresponding enums:
https://projects.blender.org/blender/blender/src/branch/main/intern/cycles/blender/addon/version_update.py#L307
I'm not too sure on how to make one other than what I see in the `compatibility.py` files in `startup/`

Reply:
We'd need to patch the loading for CyclesAttributes, rather than do anything in the renderer backend. There's a close-ish example in `startup/GafferScene/attributesCompatibility.py`, but this case might be more complicated because we want to patch `attributes.usdMIS.value` rather than a top-level plug.